build-data-capturing-gradle-samples/capture-maxparallelforks/gradle-maxparallelforks.gradle

```diff
@@ -0,0 +1,10 @@
+def buildScan = project.extensions.findByName('buildScan')
```

Comment:
I would name the files as `-max-parallel-forks` rather than `-maxparallelforks`.